src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -351,6 +362,98 @@ const generateId = () => {
         '-' + random4() + random4() + random4();
 };
 
+/*******************************************************************************
+* @description Javascript method comparable to Apex's String.format(...).
+* Replaces placeholders in Custom Labels ({0}, {1}, etc) with provided values.
+*
+* @param {string} string: Custom Label to be formatted.
+* @param {list} replacements: List of string to use as replacements.
+* @return {string} formattedString: Formatted custom label
+*/
+const format = (string, replacements) => {
```

Comment:
The format function was moved to utilCommon, let's use that version. And as a heads up - PR #4952 removes a few other functions from this file that were migrated to utilCommon.